Makefile

```diff
@@ -102,7 +107,7 @@ dependency: ## Dependency maintanance
 	go mod tidy
 
 .PHONY: dockerimage
-dockerimage: build-amd64 ## Build docker image
+dockerimage: build-amd64 plugin-yaml ## Build docker image
 	docker build --no-cache --build-arg HELM_VERSION=$(HELM_VERSION) --build-arg BUILDPLATFORM=amd64 -t $(DOCKER):$(VERSION) -f AlpineTest.Dockerfile .
```

Comment:
This is needed because of this line in AlpineTest.Dockerfile:

    COPY plugin.yaml helm-unittest/plugin.yaml

The same line is in Fedora.Dockerfile but it doesn't seem to be used anywhere (except maybe manually?). If it's still in use, I suggest we add a Make target for it as well so we can generate the plugin.yaml first.

Regardless - is this still in use? The docker build fails for me, as far as I can tell, it has nothing to do with my change:

```
Step 1/19 : FROM --platform=$BUILDPLATFORM alpine:3@sha256:1e42bbe2508154c9126d48c2b8a75420c3544343bf86fd041fb7527e017a4b4a
failed to parse platform : "" is an invalid component of "": platform specifier component must match "^[A-Za-z0-9_-]+$": invalid argument
```